elements/lisk-p2p/src/peer/inbound.ts

```diff
@@ -23,6 +23,10 @@ import {
 import { P2PDiscoveredPeerInfo } from '../p2p_types';
 
 import { SCServerSocket } from 'socketcluster-server';
+const socketErrorStatusCodes = {
+	...(SCServerSocket as any).errorStatuses,
```

Comment:
Do you think it might throw an error if `errorStatuses` is `undefined`?

Reply:
The `errorStatuses` object is always defined; it's part of the SC API.